spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -806,6 +816,10 @@ void customizeTomcat(ServerProperties serverProperties,
 			if (this.redirectContextRoot != null) {
 				customizeRedirectContextRoot(factory, this.redirectContextRoot);
 			}
+
+			if (this.tldSkipPatterns != null) {
+				factory.setTldSkip(this.tldSkipPatterns);
+			}
```

Comment:
It'd be nice to have a basic test to assert that setting that property has the desired effect. Also the key must be documented in the appendix.